tkp/distribute/serial/tasks.py

```diff
@@ -7,16 +7,16 @@
 
 
 def persistence_node_step(images, image_cache_config, sigma, f):
-    logger.info("running persistence task")
```

Comment:
Well, we also note that we're starting this step from the main logic flow:
https://github.com/transientskp/tkp/blob/ee53bc4b92e35b309182fb6047a5a658ee834855/tkp/main.py#L95
so I figured this was quite redundant